src/systems/systems/src/pressurization/mod.rs

```diff
@@ -30,26 +63,100 @@ impl Pressurization {
         }
 
         Self {
-            cpc: [CabinPressureController::new(); 2],
-            outflow_valve: PressureValve::new(),
+            cabin_pressure_simulation: CabinPressureSimulation::new(),
+            cpc: [
+                CabinPressureController::new(),
+                CabinPressureController::new(),
+            ],
+            outflow_valve: PressureValve::new_outflow_valve(),
+            safety_valve: PressureValve::new_safety_valve(),
+            residual_pressure_controller: ResidualPressureController::new(),
             active_system: active,
             landing_elevation: Length::new::<foot>(0.),
+            departure_elevation: Length::new::<foot>(0.),
             sea_level_pressure: Pressure::new::<hectopascal>(1013.25),
             destination_qnh: Pressure::new::<hectopascal>(0.),
+            is_in_man_mode: false,
+            man_mode_duration: Duration::from_secs(0),
+            packs_are_on: true,
+            lgciu_gears_compressed: true,
         }
     }
 
-    pub fn update(&mut self, context: &UpdateContext, engines: [&impl EngineCorrectedN1; 2]) {
+    pub fn update(
+        &mut self,
+        context: &UpdateContext,
+        press_overhead: &PressurizationOverheadPanel,
+        engines: [&impl EngineCorrectedN1; 2],
+        lgciu: [&impl LgciuWeightOnWheels; 2],
+    ) {
+        self.lgciu_gears_compressed = lgciu
+            .iter()
+            .all(|&a| a.left_and_right_gear_compressed(true));
```

Comment:
**thought**: `self.lgciu_gears_compressed` is not used outside of this function, and the execution order is now such that this could be a local variable.
**suggestion**: Change to a local variable:

```suggestion
        let lgciu_gears_compressed = lgciu
            .iter()
            .all(|&a| a.left_and_right_gear_compressed(true));
```

Reply:
True! Good spot.